scripts/slack_cve_report.py

```
@@ -321,7 +321,7 @@ def create_slack_message(version, results, format_type='summary', image_details=
                 "type": "header",
                 "text": {
                     "type": "plain_text",
-                    "text": f"{severity_emoji} CVE Gate Report – ACM {version}",
+                    "text": f"{severity_emoji} CVE Gate Report – MCE {version}",
```

Ruff (0.15.11)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)